ui/src/environments/environment.ts

```diff
@@ -3,5 +3,6 @@ import {testEnvironmentBase} from 'environments/test-env-base';
 export const environment = {
   ...testEnvironmentBase,
   displayTag: 'Local->Test',
-  debug: true
+  debug: true,
+  gaId: 'UA-112406425-1'
```

Comment:
Wondering if we should have a separate web property for local vs. test. Not sure it really matters...

Reply:
True - that's easy enough to do.

Reply:
Done.